server/go/go.mod

```diff
@@ -1,18 +1,18 @@
-module github.com/mlrun/log-collector
+module github.com/mlrun
```

Comment:
Shouldnt it be `github.com/mlrun/mlrun`?

Reply:
not really at this point